content/en/changelogs/1.36.0-changelog.md

```diff
@@ -45,26 +45,38 @@ controller:
 #### Breaking Changes
 
 ##### Front50 Timeout Configuration
-PR [#4773](https://github.com/spinnaker/orca/pull/4773) introduced a breaking change where timeouts were hardcoded at 10 seconds (readTimeoutMs: 10000, writeTimeoutMs: 10000, connectTimeoutMs: 10000), overriding global OkHttp timeout settings.
 
-The Kork global OkHttp defaults are:
-- connectTimeoutMs: 5000 (5 seconds)
-- readTimeoutMs: 120000 (120 seconds)
-- maxRequests: 100
-- maxRequestsPerHost: 100
-
-This hardcoding caused intermittent authentication failures during batch pipeline updates, especially with high latency or large pipeline configurations, as the global default read timeout of 120 seconds was being overridden with a much shorter 10 second timeout.
+Before PR #4773, these config properties controlled timeouts for Orca to Front50 communication:
+```yaml
+    ok-http-client:
+      connectTimeoutMs: 5000       # Default: 5 seconds
+      readTimeoutMs: 120000        # Default: 120 seconds (2 minutes)
+      maxRequests: 100             # Default: 100
+      maxRequestsPerHost: 100      # Default: 100
+```
+After PR #4773, the config properties changed to:
+```yaml
+    front50:
+      okhttp:
+        connectTimeoutMs: 10000    # Hardcoded default: 10 seconds
+        readTimeoutMs: 10000       # Hardcoded default: 10 seconds
+        writeTimeoutMs: 10000      # Hardcoded default: 10 seconds
+```
+Spinnaker installations that set the old config properties (`ok-http-client`) see a change in behavior in version 1.36.0. The old properties are ignored, and the new hardcoded defaults (10 seconds) are used instead.
```

Comment:
Even users that didn't set any ok-http-client properties see a change in behavior.

Reply:
Correct, since the new default is shorter

Reply:
Yeah need to tweak the text to allow for this.  Probably rephrased to
* in 1.36, there's a new set of defaults that replace the default timeouts on internal calls to front50 with drastically lower defaults.  Further these defaults now ignore any overridden okhttp settings for inter-service communications.  This can break certain operations that might have previously worked due to the lower timeout defaults.  The most common case is when service accounts are created as part of saving a pipeline.  As such, you should adjust your front50 timeout calls explicitly for your use case using these different properties.

Reply:
Done in https://github.com/spinnaker/spinnaker.io/pull/504/commits/9f451c38ba03007088f208b8cce77cdfcf6fd2e3.  I'd like to reference https://github.com/spinnaker/orca/pull/4851 and be able to say when we've at least restored the timeout behavior, but let's not do that until it's actually merged.  It still needs some work from my perspective.